tests/bootstrap.php

```diff
@@ -1,24 +1,16 @@
 <?php
 
-/**
- * @license GNU GPL v2+
- * @since 1.0
- *
- * @author mwjames
- */
+declare( strict_types = 1 );
 
-error_reporting( E_ALL | E_STRICT );
-date_default_timezone_set( 'UTC' );
-
-if ( PHP_SAPI !== 'cli' && PHP_SAPI !== 'phpdbg' ) {
+if ( PHP_SAPI !== 'cli' ) {
```

Comment:
`&& PHP_SAPI !== 'phpdbg'` should remain so it can run with the internal PHP debugger